resources/components/ImageCard.vue

```diff
@@ -331,4 +367,27 @@ module.exports = {
 		margin-left: auto;
 	}
 }
+
+.wbmad-custom-tag-button.mw-button {
```

Comment:
Some of these styles are covered by the base button and aren't needed:
- transition
- background-color
- border
- color
- cursor

Can we move border-radius up in the order so rules are alphabetical? We haven't discussed rule this as a team but I have a Very Strong Opinion on this 😅